src/style/index.scss

```diff
@@ -5,12 +5,11 @@
   box-sizing: border-box;
   flex-direction: row;
   flex-wrap: wrap;
-  @include clearfix();
```

Comment:
@jsmuse can you fix that specifically for the space-between and not for all the case?
@abraztsov why we need a clearfix here?

Reply:
@jsmuse @alexeybondarenko we don't. let's revert all changes except this one https://github.com/abraztsov/ReactSimpleFlexGrid/pull/8/commits/4a49e6eb93b853d23bda39c60be5779064461acc (delete clearfix) and I'll merge request